02LessonTwo/LessonTwo.js

```diff
@@ -5,10 +5,28 @@ module.exports = function() {
     // should return "cat" when character is 'garfield'
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
+    if (character === 'scooby') {
+    	   return "dog";
+        }
+    	else if (character === 'garfield') {
+    		return "cat";
+    	}
+    	else if (character === 'nemo') {
+    		return "fish";
+    	}
+    	else {
+    		return false;
+    	}
   }
 
   this.isEven = function (number) {
     // should return true is number is even (divisible by 2)
-  }
+    if (number % 2 === 0) {
+    	return true;
+        }
+        else {
+            return false;
+        }
+    }
 
```

Comment:
See if you can condense lines 24-30 into a single line. Ask yourself what kind of value does the expression `number  % 2 === 0` result in.